actors/migration/nv8/top.go

```diff
@@ -0,0 +1,79 @@
+package nv7
+
+import (
+	"context"
+
+	address "github.com/filecoin-project/go-address"
+	"github.com/filecoin-project/go-state-types/abi"
+	"github.com/ipfs/go-cid"
+	cbor "github.com/ipfs/go-ipld-cbor"
+	"golang.org/x/xerrors"
+
+	"github.com/filecoin-project/specs-actors/v3/actors/builtin"
+	"github.com/filecoin-project/specs-actors/v3/actors/states"
+	"github.com/filecoin-project/specs-actors/v3/actors/util/adt"
+)
+
+// Config parameterizes a state tree migration
+type Config struct{}
+
+func DefaultConfig() Config {
+	return Config{}
+}
+
+type MigrationInfo struct {
+	address    address.Address // actor's address
+	balance    abi.TokenAmount // actor's balance
+	priorEpoch abi.ChainEpoch  // epoch of last state transition prior to migration
+}
+
+type StateMigrationResult struct {
+	NewHead  cid.Cid
+	Transfer abi.TokenAmount
```

Comment:
This was quite a specialized artifact of the v4 migration and I think we can retire it until we need it again if ever.  Keeping the wrapping struct is nice to make extension beyond just returning the head really easy.